### ISSUE TITLE
Compress json to zip

### ISSUE THREAD

**Issue by sawka6630** [tmo1/sms-ie#191]
Please add post-export zip compression of json files (created by auto backup feature particularly). It will reduce the target file size significantly.
For example:

- contacts.json file comes from 15.51 Mb to 673 kb when compressed using regular (level 6) compression and to 592 kb when compressed using the highest (level 9) compression,
- calls.json file comes from 2.69 Mb to 109 kb (level 6) and to 99 kb (level 9).

In general this will allow to save ~17.5 Mb per day.

**Comment by tmo1** [comment]
I agree that the space savings would be desirable. If and when we switch the format from simple JSON to (ND)JSON encapsulated in a ZIP file, then we'll get compression automatically (as is currently the case for messages), but format changes require careful consideration and planning, since they break backward compatibility.
